### PR TITLE
UserGuide: small edit

### PR DESCRIPTION
**Pull request by CindyTsai1**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -59,7 +59,7 @@ Format: `help`
 ==== Adding a person manually
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [b/BIRTHDAY] [tt/TIMETABLE_URL] [t/TAG] ...` +
+Format: `add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [tt/TIMETABLE_URL] [t/TAG] ...` +
```
Comment (nbriannl) [inline docs/UserGuide.adoc]: alot of people conflicting over this line. putting your PR on hold

**Review by April0616 — DISMISSED** [review]
Correct

File: docs/UserGuide.adoc (hunk 1 above)
Comment (CindyTsai1) [reply docs/UserGuide.adoc]: I believe we cannot expect all contacts that we add to have phone numbers
we might only have their emails as well
so it's more logical to keep all fields optional while the name is a compulsory field

Comment (CindyTsai1) [reply docs/UserGuide.adoc]: If we all agree on this, the addCommand.java will have to be altered

File: docs/AboutUs.adoc [hunk 2]
```diff
@@ -11,8 +11,8 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 == Project Team
 
 === Fan Yuting
-image::yuting.JPG[width="150", align="left"]
-{empty}[https://github.com/April0616[github]] [<<April0616#, portfolio>>]
+image::damithc.jpg[width="150", align="left"]
```
Comment (zacharytang) [inline docs/AboutUs.adoc]: Something wrong here? Why is the file name reverted back to the dummy file

**Review by nbriannl — CHANGES_REQUESTED** [review]
change line 14 in AboutUs.adoc back to yuting.jpg

**Review by nbriannl — APPROVED** [review]

**Pull request merged**